config.mainnet.ts

```diff
@@ -3,4 +3,6 @@ import { NETWORK_MAIN } from './src/lib/Constants';
 export default {
     keyguardEndpoint: 'https://keyguard.nimiq.com',
     network: NETWORK_MAIN,
+    networkEndpoint: 'https://network.nimiq.com',
+    privilegedOrigins: [ 'safe.nimiq.com' ],
```

Comment:
An origin includes the protocol, so I think this should be `https://safe.nimiq.com`.